contracts/PredictionMarketSystem.sol

```diff
@@ -55,31 +55,43 @@ contract PredictionMarketSystem is OracleConsumer, ERC1155 {
         uint payout
     );
 
+    
     /// Mapping key is an condition ID. Value represents numerators of the payout vector associated with the condition. This array is initialized with a length equal to the outcome slot count.
-    mapping(bytes32 => uint[]) public payoutNumerators;
+    /// e.g Condition with 3 outcomes [A, B, C] and two of those correct [0.5, 0.5, 0]
+    /// In Ethereum there are no decimal values so, to represent 0.5 it has to be represented by a division of two intengers 1/2 == 0.5. That's why we need numerator and denominator values.
+    mapping(bytes32 => uint[]) public payoutNumerators; 
     mapping(bytes32 => uint) public payoutDenominator;
 
+    // Note:
+    /// Numerator is also used as a check of initialization. If numerator is null, means condition was not created/prepared.
+    /// Denominator is used for checking if the condition has been resolved.
```

Comment:
Please move this note to before the `mapping` declarations for the payout fields. Comments preceded by `///` get parsed into the documentation.